static/styles/zulip.css

```diff
@@ -2053,7 +2053,7 @@ div.focused_table {
     font-size: 19px;
     text-align: center;
 
-    &.fa-group {
+    &.fa {
```

Comment:
While this produces correct results, I think this is really confusing, since the reason we're adding this margin is not because there's a font-awesome icon present.  I think it'd be better to add a new class name that we apply to the relevant elements and have these margins be based on the presence of that class.

Also, I think this is a good PR to practice this on -- each of the 3 changes in this PR are independent improvements fixing different bugs, and the cleanest writing of this would be to have each in their own commit.  Can you split this?